src/main/app/drafts/tasks/claimAmount.ts

```diff
@@ -17,12 +17,12 @@ export class ClaimAmount {
     if (claim.interestType.option === InterestTypeOption.SAME_RATE) {
       if (claim.interestDate.type === InterestDateType.SUBMISSION) {
         return claim.amount.totalAmount() > 0 &&
-          claim.amount.totalAmount() < 10000 &&
+          claim.amount.totalAmount() <= 10000 &&
           claim.interestType.isCompleted() &&
           claim.interestRate.isCompleted()
       } else {
         return claim.amount.totalAmount() > 0 &&
-          claim.amount.totalAmount() < 10000 &&
```

Comment:
Is it worth having hardcoded 10000 a constant somewhere?

Reply:
its already a constant else where, I can refactor it, but I just did the minimum change here